app/controllers/order_controller.rb

```diff
@@ -0,0 +1,112 @@
+class OrderController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_action :user_params, only: :create
+  before_action :load_orders, only: :update
+
+  def index; end
+
+  def create
+    @order = current_user.orders.new name: user_params[:name],
+      phone: user_params[:phone], address: user_params[:address]
+
+    @current_cart = current_cart
+    @current_quantity = current_quanity
+    @errorlist = []
+
+    # Check inventory of current product
+    if @current_cart.nil?
+      respond_to do |format|
+        format.js{render "empty.js.erb"}
+      end
+    else
+      test_invetory
+      errorlist_check
+    end
+  end
+
+  def update
+    @orders.update_attribute :status, params[:status].to_i
```

Comment:
hỏi ngu tí, update mà false thì vẫn response to js đúng ko em

Reply:
Dạ :3 xét thêm trường hợp false nữa chị format.js nhưng file js khác